index.html

```diff
@@ -1,16 +1,269 @@
+  </head>
+    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="none" data-md-color-accent="none">
+    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
+    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
+    <label class="md-overlay" for="__drawer"></label>
+    <div data-md-component="announce">
+
+    </div>
+      <div data-md-component="outdated" hidden>
+        <aside class="md-banner md-banner--warning">
+        </aside>
+      </div>
+
+<header class="md-header" data-md-component="header">
+  <nav class="md-header__inner md-grid" aria-label="Header">
+    <div class="md-header__title" data-md-component="header-title" width="100%">
+      <div align="center">
+        <span class="md-ellipsis">
+          Eclipse iceoryx™ is an inter-process-communication middleware that enables virtually limitless data transmissions at constant time
+        </span>
+      </div>
+    </div>
+  </nav>
+</header>
+
+    <div class="md-container" data-md-component="container">
+      <main class="md-main" data-md-component="main">
+        <div class="md-main__inner md-grid" width="100%">
+          <div class="md-content" data-md-component="content"  align="center">
+            <article class="md-content__inner md-typeset" style="max-width: 1000px">
+
+<p align="center">
+    <img src="https://user-images.githubusercontent.com/8661268/114321508-64a6b000-9b1b-11eb-95ef-b84c91387cff.png" width="50%">
+
+
+<table align="center">
+<thead>
+<tr>
+<th><h2>iceoryx</h2></th>
+<th><h2>iceoryx2</h2></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td width="50%">
+  <ul>
+    <li>C++</li>
+    <li>True zero-copy</li>
+    <li>Central daemon</li>
+    <li>Multiple messaging patterns
+      <ul>
+        <li>publish-subscribe</li>
+        <li>request-response</li>
+      </ul>
+    </li>
+    <li>Events</li>
+    <li>Wide range of platforms
+      <ul>
+        <li>Linux</li>
+        <li>Windows</li>
+        <li>macOS</li>
+        <li>FreeBSD</li>
+        <li>QNX</li>
+        <li>FreeRTOS</li>
+      </ul>
+    </li>
+    <li>Some language bindings
+      <ul>
+        <li>C</li>
+        <li>Rust (partial)</li>
+      </ul>
+    </li>
+    <li>Host2Host communication via DDS</li>
+  </ul>
+</td>
+<td>
+  <ul>
+    <li>Rust</li>
+    <li>True zero-copy</li>
+    <li>No central daemon</li>
+    <li>Multiple messaging patterns
+      <ul>
+        <li>publish-subscribe</li>
+        <li>request-response</li>
+        <li>pipeline (planned)</li>
+        <li>blackboard (planned)</li>
+      </ul>
+    </li>
+    <li>Events</li>
+    <li>Wide range of platforms
+      <ul>
+        <li>Linux</li>
+        <li>Windows</li>
+        <li>macOS</li>
+        <li>FreeBSD</li>
+        <li>Android (planned)</li>
```

Comment:
@elBoberido What do you think about only adding features to the feature list when they are implemented? I think the list becomes more readable and easier to grasp then.

Reply:
Sure, what do you think about consolidating the points into one bullet point. IMO the main target of this landing page are newcomers and we should not confuse them too much with details, but provide them with a good overview and comparison of what the two versions offer. What do you think?

Reply:
You mean just `Wide range of platforms`? That doesn't give many information. The list isn't that long and gives a nice overview

Reply:
No, I meant the following:

- Written in Rust
- Uses true zero-copy inter-process communication
- No central daemon is needed
- Supports multiple messaging patterns
  - publish-subscribe
  - request-response
- Events
- Wide range of platforms
   - Linux
   - Windows
   - macOS
   - FreeBSD
- Many features are planned e.g. additional platforms, see [here](http://foo.bar)

No hard feelings on this, though. Just a suggestion. Up to you.